mm2src/coins/eth.rs

```diff
@@ -545,7 +545,7 @@ async fn get_raw_transaction_impl(coin: EthCoin, req: RawTransactionRequest) ->
     })
 }
 
-async fn withdraw_impl(ctx: MmArc, coin: EthCoin, req: WithdrawRequest) -> WithdrawResult {
+async fn withdraw_impl(_ctx: MmArc, coin: EthCoin, req: WithdrawRequest) -> WithdrawResult {
```

Comment:
As `ctx` is not used anymore, please remove it from args.

Reply:
ctx: MmArc was removed